src/languages/es.ts

```diff
@@ -1808,6 +1809,9 @@ export default {
                 subtitle: 'Añade una etiqueta para realizar el seguimiento de proyectos, ubicaciones, departamentos y otros.',
             },
         },
+        taxes: {
+            subtitle: 'Añada nombres de impuestos, tipos impositivos y establezca valores por defecto.',
+        },
```

Comment:
Are translations verified by someone?

Reply:
Waiting for confirmation

Reply:
https://expensify.slack.com/archives/C01GTK53T8Q/p1709801973344319